functions/anonymous-functions.md

```diff
@@ -0,0 +1,198 @@
+# Anonymous Functions
+
+## Introduction
+
+Yet another permutation of the function in the JavaScript language!
+
+Anonymous functions are a useful way to define functions in the context they're used. This usage can be more readable sometimes, especially when it's more convenient to read a function's definition immediately over chase its definition, which may be in a different file.
+
+## Syntax
+
+An anonymous function is a function created with any valid syntax, but it is not referenced by a variable.
+
+Instead, an **anonymous function** is a function defined in the one place that it's used.
+
+Technically, many of the methods in objects we created are anonymous functions!
+
+<!-- prettier-ignore-start -->
+```js
+const myTedTalk = {
+  title: 'Inspiration From Inspiring Things',
+  summary: 'A story about where inspiration comes from.',
+  summarize: function() {
+    return `${this.title}: ${this.summary}`;
+  }
+}
+
+myTedTalk.summarize();
+```
+<!-- prettier-ignore-end -->
+
+We never _named_ the value of the `summarize` property, or referenced it in a variable. We defined the function in the one place where it's needed, anonymously!
+
+## Uses for Anonymous Functions: `forEach` Loops
+
+An anonymous function is useful when create for-each loops.
+
+The `forEach` loop in JavaScript is a function that operates off of a collection (like an array). It takes in one argument: a **function**.
+
+_The passed-in function_ takes in one argument, which represents one item from the collection.
```

Comment:
Let's be honest. `forEach` is awful and should be minimized. It's a convenient place to start talking about anonymous functions, but I'd still like to include some verbiage to subtly indicate that this isn't necessarily (or ever! 😆 ) the best way to do this.

Also, `forEach` is an array function. If we want the flexibility of other collections, that's a case for using a for/in or for/of loop.

```suggestion
An anonymous function is useful to create a certain kind of for-each loop.

The `forEach` method in JavaScript is defined on arrays. It takes in one argument: a **function**.

_The passed-in function_ also takes in one argument, which represents one item from the collection.
```